### PR TITLE
Centralize actions and constants for better scalability

### PR DESCRIPTION
**Pull request by hajjiTarik**

### CODE REVIEW

File: src/updater.js [hunk 1]
```diff
@@ -78,7 +79,7 @@ const offlineUpdater = function offlineUpdater(
   }
 
   // Remove completed actions from queue (success or fail)
-  if (action.meta != null && action.meta.completed === true) {
+  if (action.meta !== null && action.meta.completed === true) {
```
Comment (migueloller) [inline src/updater.js]: `action.meta != null` and `action.meta !== null` do very different things. The former checks for `undefined` while the latter doesn't. Given that the scope of this PR is for making action types constants, is there any reason why this change is here?

Comment (hajjiTarik) [reply src/updater.js]: Ah sorry i have not paid attention, i will fix it
